src/pybind/mgr/dashboard/frontend/src/app/ceph/block/rbd-form/rbd-form.component.ts

```diff
@@ -443,6 +480,26 @@ export class RbdFormComponent implements OnInit {
     return request;
   }
 
+  cloneRequest(): RbdFormCloneRequestModel {
+    const request = new RbdFormCloneRequestModel();
+    request.child_pool_name = this.rbdForm.get('pool').value;
+    request.child_image_name = this.rbdForm.get('name').value;
+    request.obj_size = this.formatter.toBytes(this.rbdForm.get('obj_size').value);
+    if (!this.defaultFeaturesFormControl.value) {
```

Comment:
Nit: when I cloned an image w/ default features, the clone model didn't have the default features checkbox selected but did have each individual feature selected.

Reply:
I've now created a new REST API endpoint to get the default features from the "rbd_default_features" config, which allowed me to remove the "Default features" checkbox from the UI.

Default features will now be pre-selected on the creation form, which I believe is an usability improvement.